include/cuco/detail/hyperloglog/hyperloglog_impl.cuh

```diff
@@ -402,7 +402,7 @@ class hyperloglog_impl {
    *
    * @return Approximate distinct items count
    */
-  [[nodiscard]] __host__ constexpr size_t estimate(cuda::stream_ref stream) const
+  [[nodiscard]] __host__ size_t estimate(cuda::stream_ref stream) const
```

Comment:
I wonder why this can be `constexpr` with `cudaMemcpyAsync` inside? Is `cudaMemcpyAsync` also `constexpr`?

Reply:
yeah I don't know either. cannot get a godbolt reproducer thus there must be something missing in this case.